packages/headless/src/api/analytics/commerce-analytics.test.ts

```diff
@@ -0,0 +1,35 @@
+import {buildMockProductListingV2State} from '../../test/mock-product-listing-v2-state';
```

Comment:
I considered entirely removing the analytics part for now since the commerce sub-package is likely going to end up using the new analytics protocol and/or server-side events.

However, after discussing it with @Spuffynism , we decided to keep it for now since the product listing V2 endpoint requests are translated into Search API queries anyways, so it's not like the current way of sending analytics events makes no sense either. It's just that it's going to be replaced with something else in the near future.